apps/jobs/views.py

```diff
@@ -65,12 +65,16 @@
 from .tasks import download_file_and_publish_submission_message
 from .utils import (
     calculate_distinct_sorted_leaderboard_data,
-    get_submission_model,
-    get_remaining_submission_for_a_phase,
     handle_submission_rerun,
     is_url_valid,
 )
 
+from .utils import (
```

Comment:
@vkartik97 why are we adding redundant imports for `from .utils import` can't we add new methods to existing import?

Reply:
I must have been done while resolving merge conflicts. Thanks for pointing it out